src/directives/bind.js

```diff
@@ -68,8 +68,8 @@ export function handleAttributeBindingDirective(component, el, attrName, express
     } else {
         attrName = modifiers.includes('camel') ? camelCase(attrName) : attrName
 
-        // If an attribute's bound value is null, undefined or false, remove the attribute
-        if ([null, undefined, false].includes(value)) {
+        // If an attribute's bound value is null, undefined or false and does not include aria, remove the attribute
+        if ([null, undefined, false].includes(value) && !attrName.includes('aria')) {
```

Comment:
You would likely want `!attrName.startsWith('aria')`, right? Otherwise you'd match on attributes containing `aria` inadvertently, like `data-variable`

Reply:
Done.

Reply:
Since the prefix is `aria-` maybe better to include the `-` as well?